src/status_im/navigation/core.cljs

```diff
@@ -438,4 +439,6 @@
                               :community
 
                               :else
-                              :home))}))
+                              :home))
+     :dispatch-n (enc/conj-when []
+                                (and (= view-id :chat) [:chat.ui/sync-all-deleted-for-me-messages]))}))
```

Comment:
Get all deleted-for-me messages that not yet synced with status-go and sync them